src/devices/cpu/i386/i486ops.hxx

```diff
@@ -284,7 +284,7 @@ void i386_device::i486_group0F01_16()      // Opcode 0x0f 01
 			}
 		case 6:         /* LMSW */
 			{
-				uint16_t b;
+				uint16_t b = 0;
```

Comment:
Why would this be undefined?  b is set on both sides of the if statement.

Reply:
As far as I can tell, the definition of `FAULT` doesn't use `b`.  Does it shut up the warning if you move the declaration of `b` after the GPF check?